integration/kubernetes/k8s-replication.bats

```diff
@@ -26,8 +26,8 @@ setup() {
 	kubectl create -f "${pod_config_dir}/test-replication-controller.yaml"
 
 	# Check replication controller
-	kubectl describe replicationcontrollers/"$replication_name" | \
-		grep "replication-controller"
+        kc_rc_chk="kubectl describe replicationcontrollers/$replication_name | grep replication-controller"
```

Comment:
What does `kc_rc_chk` stands for?
I'd go for a more descriptive variable name here, @c3d.

Apart from this, LGTM.

Reply:
or just call that `cmd` as in many other scripts. Ah, and make it a `local` variable :)

Reply:
Done. `cmd` is actually used a few lines below.